### PR TITLE
Added schema for python2packages for azure automation

### PR DESCRIPTION
**Pull request by D1v38om83r**
Added schema for python2packages for azure automation. Also updated old schema for 2017-05-15-preview\Microsoft.Automation.json

### CODE REVIEW

File: schemas/2014-04-01-preview/deploymentTemplate.json [hunk 1]
```diff
@@ -269,6 +269,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-05-15-preview/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_sourceControls_sourceControlSyncJobs" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-01-15/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_compilationjobs" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-01-15/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_nodeConfigurations" },
+				  { "$ref": "https://schema.management.azure.com/schemas/2018-06-30/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_python2Packages" },
```
Comment (vrdmr) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: nit: Indentation.

File: schemas/2015-01-01/deploymentTemplate.json [hunk 2]
```diff
@@ -292,6 +292,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-05-15-preview/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_sourceControls_sourceControlSyncJobs" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-01-15/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_compilationjobs" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-01-15/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_nodeConfigurations" },
+				  { "$ref": "https://schema.management.azure.com/schemas/2018-06-30/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_python2Packages" },
```
Comment (vrdmr) [inline schemas/2015-01-01/deploymentTemplate.json]: nit: Indentation

**Comment by D1v38om83r** [comment]
@Tiano2017 request for code review

File: schemas/2014-04-01-preview/deploymentTemplate.json (hunk 1 above)
Comment (TianoMS) [inline schemas/2014-04-01-preview/deploymentTemplate.json]: >				 [](start = 0, length = 4)

please use spaces.

File: tests/2018-06-30/Microsoft.Automation.tests.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+    "tests": [
+                {
```
Comment (TianoMS) [inline tests/2018-06-30/Microsoft.Automation.tests.json]: >                 [](start = 0, length = 16)

please use consistent indentation: 2 spaces.

**Comment by Tiano2017** [comment]
PR looks good. please fix the indentation issues.

File: schemas/2014-04-01-preview/deploymentTemplate.json (hunk 1 above)
Comment (D1v38om83r) [reply schemas/2014-04-01-preview/deploymentTemplate.json]: fixed in new commit

Comment (D1v38om83r) [reply schemas/2014-04-01-preview/deploymentTemplate.json]: fixed in new commit

**Comment by D1v38om83r** [comment]
Fixed indentation issue in new commit

**Pull request merged**